graylog2-server/src/main/java/org/graylog2/rest/resources/search/SearchResource.java

```diff
@@ -144,6 +150,26 @@ protected SearchResponse buildSearchResponse(SearchResult sr, org.graylog2.index
                 timeRange.getTo());
     }
 
+    protected Set<IndexRangeSummary> indexRangeListToValueList(Set<IndexRange> indexRanges) {
+        final Set<IndexRangeSummary> result = Sets.newHashSetWithExpectedSize(indexRanges.size());
+
+        for (IndexRange indexRange : indexRanges) {
+            result.add(IndexRangeSummary.create(indexRange.getIndexName(), indexRange.getCalculatedAt(), indexRange.getStart(), indexRange.getCalculationTookMs()));
+        }
+
+        return result;
+    }
+
+    protected List<ResultMessageSummary> resultMessageListtoValueList(List<ResultMessage> resultMessages) {
+        final List<ResultMessageSummary> result = Lists.newArrayListWithExpectedSize(resultMessages.size());
```

Comment:
Why not [`Lists#newArrayListWithCapacity(int)`](http://docs.guava-libraries.googlecode.com/git/javadoc/com/google/common/collect/Lists.html#newArrayListWithCapacity(int)) or simply `new ArrayList<>(int)`?